tools/test_containers_compose

```diff
@@ -20,7 +20,12 @@ wait_until() {
 
 setup_containers() {
   for retry in {2..0}; do sudo docker-compose build && break; echo "Remaining retries $retry"; done
-  sudo MOJO_CLIENT_DEBUG=1 docker-compose up -d
+  sudo MOJO_CLIENT_DEBUG=1 docker-compose up -d || exit_code=$?
+  if [[ -n $exit_code ]]; then
```

Comment:
Now we're getting this error message:
```
tools/test_containers_compose: line 24: exit_code: unbound variable
```
Since the script does `set -u`, variables have to be set in every case (even if it's just an empty string).
So I suggest a
```
exit_code=0
```
before that line.

Reply:
Done, thanks.